index.html

```diff
@@ -81,6 +81,176 @@ <h2>Glad to see you!</h2>
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <a href="#" class="work-btn">See Project</a>
+            </div>
+          </div>
+          <div class="card card-item-3">
+            <div class="card-work">
+              <img src="./images/gymfit.svg" alt="gymfit" />
+              <h2>Tonic</h2>
+              <div class="card-caption">
+                <span>Canopy</span>
+                <img src="./images/Counter.svg" alt="counter" />
+                <span>Back End Dev</span>
+                <img src="./images/Counter.svg" alt="counter icon" />
+                <span>2015</span>
+              </div>
+              <div class="work-text">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+              <div class="stacks">
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <a href="#" class="work-btn">See Project</a>
+            </div>
+          </div>
+          <div class="card card-item-4">
+            <div class="card-work">
+              <img src="./images/art-printing.svg" alt="Art printing" />
+              <h2>Multi-Post Stories</h2>
+              <div class="card-caption">
+                <span>Canopy</span>
+                <img src="./images/Counter.svg" alt="counter" />
+                <span>Back End Dev</span>
+                <img src="./images/Counter.svg" alt="counter icon" />
+                <span>2015</span>
+              </div>
+              <div class="work-text">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+              <div class="stacks">
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <a href="#" class="work-btn">See Project</a>
+            </div>
+          </div>
+        </section>
+      </div>
+      <div class="about-wrapper">
+        <section class="headline">
+          <div class="about">
+            <h2>About <span>Myself</span></h2>
+          </div>
+
+          <div class="about-desc">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <div class="social">
+            <a href="#" class="about-btn">LET’S CONNECT</a>
```

Comment:
@keneogu I'm really confused right now, one reviewer told me to change the title to either a link or button and you said I should change it to hx what I do now because this is my last code review. See attached file for the comment of the previous code reviewer.
![connects](https://user-images.githubusercontent.com/45001916/143627347-433645b9-af7d-411c-9654-3b7ffb94f1ca.png)